Elements/src/Geometry/Line.cs

```diff
@@ -1045,6 +1045,30 @@ public bool TryGetOverlap(Line line, out Line overlap)
             return true;
         }
 
+        /// <summary>
+        /// Creates new line with verticies of current and joined line
+        /// </summary>
+        /// <param name="line">Collinear line</param>
+        /// <returns>New line containing verticies of all merged lines</returns>
+        /// <exception cref="ArgumentException">Throws exception when lines are not collinear</exception>
+        public Line MergeCollinearLine(Line line)
```

Comment:
Can we call this `MergedWithCollinear`. In Elements we use the past tense for methods that result in a new object.

Reply:
done [here](https://github.com/hypar-io/Elements/pull/810/commits/131f95956f60df5300cff98d5738e98a32623a57)